### ISSUE TITLE
Adding new training images with pretrained model

### ISSUE THREAD

**Issue by doublejtoh** [JiahuiYu/generative_inpainting#37]
When trying to resume training new images based on pretrained model(restored model), do i have to train with not only new images but also previous trained images or with only new images?
In the former case, i wonder if it can cause overfitting with previous trained images as training iteration increases.

**Comment by JiahuiYu** [comment]
Either case is fine. It is unlikely to overfit the places2 dataset for image generation tasks.

**Comment by KangSH9776** [comment]
hello!

We did not get good results when We trained with new 1000 images, without changing the hyper-parameters
We are trying to fine-tune it using pre-train model.

When trying to resume training new images based on pretrained model(restored model), do i have to train with not only new images but also previous trained images or with only new images?

I want to fine tune 1,000 additional images in the pre-trained model with places dataset

Can we initialize the weight value of pre-train model to weight value of train?
By initializing the weight value of the 'training' to the weight value of the pre-trained model, is it possible to do additional training?
Is there another way to 'train' with the information value of  'pre-train model'?

thank you
